lib/events/s3sessions/s3handler.go

```diff
@@ -97,6 +97,8 @@ type Config struct {
 	Insecure bool
 	// DisableServerSideEncryption is an optional switch to opt out of SSE in case the provider does not support it
 	DisableServerSideEncryption bool
+
+	UsePathStyle *bool
```

Comment:
Add godoc, and also per my comment above I think `*bool` is awkward, I would invert the flag so the default is false. I don't see other parameters here using this pattern.